### ISSUE TITLE
[Enhancement] List sub sections

### ISSUE THREAD

**Issue by lydemann** [kirbydesign/designsystem#1604]
**Please add a short description of your enhancement request**
Support for creating list sub section, which is a section in a section, containing 1 or more sections that are separated from other items in the section.

Eg. looks like this, with 8 px separation in this case for grouping an item with an item containing action buttons:
![image](https://user-images.githubusercontent.com/9210691/120981840-5b7f3b80-c778-11eb-9f2c-94b7709e5b34.png)

**Describe the solution you'd like**
Create a subSection input on items in a list, for making them grouped as a subsection.

**Describe alternatives you've considered**

**Additional context**

### Kick Off:

- [ ] Ensure the enhancement is `UX refined` and aligned with UX 
_The component and/or enhancement should be published and available in the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)_
- [ ] Ensure the enhancement has been `Tech refined` with @kirbydesign/kirby-guild and this issue is updated with a clear implementation description  
_This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation)_
- [ ] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code:

- [ ] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
- [ ] Create a draft implementation and push to Github
- [ ] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request
- [ ] Implement unit tests
- [ ] Update Cookbook Examples and Showcase, i.e. see [Radio](https://cookbook.kirby.design/home/showcase/radio)  
_Also remember to add any relevant new API documentation_

### Review:

- UX review:
  - [ ] Ensure implementation is correct in relation to the UX design and the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)
  - [ ] With UX agree on the version of the implementation
- Code review:
  - [ ] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review  
_Remember to add `closes #issueno` to the description of the PR._
  - [ ] Once approved, merge feature branch/PR to master
- [ ] Ask a member of @kirbydesign/kirby-guild to add a link to component showcase from Kirby Component Status and update the version number

:tada: Celebrate

**Comment by MadsBuchmann** [comment]
@lydemann It would of course be ideal if we could solve this for the virtual scroll- and the normal list. But as the virtual scroll list is still a work in progress, i would like to ask, if we would solve anything for you now by first introducing this to the normal list? Then the virtual scroll list can be considered at a later time in a separate issue.

**Comment by lydemann** [comment]
Introducing this to the normal list, for now, is fine as we are not using the virtual scroll list yet anyways.

**Comment by mark-drastrup** [comment]
We have talked to the team who created this issue to figure out how to flag items as a being sub section. We agreed that items in the items array should have a flag eg. `isStandAlone: true`, which is then used to apply the necessary classes.
